### DISCUSSION TITLE
Updating Microsoft Graph PowerShell Modules for Azure Automation

### DISCUSSION THREAD

**Discussion by joshuaflickinger** [community/community#124402] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I have an upgraded version of your code for this automation to now support updating both Azure Runbook powershell versions 5.1 and 7.2. How can I merge these changes into your published version.

Tried to create a pull request but I am not a collaborator.

[link]

**Comment by ebndev** [comment]
Hi @joshuaflickinger,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from a specific project. You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by joshuaflickinger** [comment]
Hey @ebndev,

I am not asking for support but am instead sharing an updating version of one of the programs here. Microsoft made changes and is pushing everyone to start using Powershell 7.2, and the code in it's current state defaults to only updating Powershell 5.1, but with the changes I made it now does both 5.1 and 7.2.

Thank you!

**Reply by ebndev** [reply]
I see, thanks for taking initiative to do that.

Microsoft and the GitHub product team will unlikely see this message but I will reopen this discussion in case other users would like to download the script.

Can you edit your post to include the link to the GitHub repo this file is in? We don't allow files that automatically download in Discussions.

Thanks!

**Reply by joshuaflickinger** [reply]
Understood, thanks for the help!

Here's a link to the GitHub repo of the new version I made that includes the fix: https://github.com/joshuaflickinger/MicrosoftScripts/blob/main/UpdateAzureAutomationGraphModules5.1and7.2.ps1

In case it's helpful, here's a link to the original that I edited: https://github.com/12Knocksinna/Office365itpros/blob/master/Update%20AzureAutomationGraphModules.PS1

Thank you!
